packages/react-components/react-charts-preview/src/components/DonutChart/DonutChart.tsx

```diff
@@ -325,16 +322,18 @@ export const DonutChart: React.FunctionComponent<IDonutChartProps> = React.forwa
             </svg>
           </div>
         </div>
-        <Popover positioning={{ target: virtualElement }} open={isPopoverOpen} openOnHover>
-          <PopoverSurface tabIndex={-1}>
-            <ChartHoverCard
-              Legend={xCalloutValue ? xCalloutValue : legend}
-              YValue={yCalloutValue ? yCalloutValue : value}
-              color={color}
-              culture={props.culture}
-            />
-          </PopoverSurface>
-        </Popover>
+        <div id={_calloutId}>
```

Comment:
can you assign the id directly to popover

Reply:
Previously, the callout element was using an id, but presently the popover element operates without an id, resulting in the failure of a particular testcase. Now I have removed this id and changed the testcase.